test/test_public_bindings.py

```diff
@@ -210,6 +210,7 @@ def test_no_new_bindings(self):
             "TupleType",
             "Type",
             "unify_type_list",
+            "UnionType",
```

Comment:
Do we really want this to be in `torch.`?

Reply:
You need to add documentation to this if it is public API on `torch.` no?

Reply:
cc @ansley can you comment?

Reply:
Thank you all for bringing this to my attention! I missed Alban's first comment, and the second one came after I had shipped the diff. I didn't want to revert it because it was high priority to get it out before the branch cut. I'm actively working on a cleanup diff now

Reply:
Thanks!